src/uct/api/tl.h

```diff
@@ -162,8 +162,7 @@ typedef struct uct_iface_ops {
 
     ucs_status_t (*ep_create_sockaddr)(uct_iface_h iface,
                                        const ucs_sock_addr_t *sockaddr,
-                                       uct_sockaddr_conn_reply_callback_t reply_cb,
-                                       void *arg, uint32_t cb_flags,
+                                       uint32_t cb_flags,
```

Comment:
should remove cb_flags as well

Reply:
and always use UCT_CB_FLAG_ASYNC in the transport?
we are not planning to support UCT_CB_FLAG_SYNC at all?

Reply:
but if there is no callback on client side, why even need to have a mode?
should keep it only for server side

Reply:
right! there is no cb anymore :)